website/docs/cli/commands/providers/lock.mdx

```diff
@@ -88,6 +88,11 @@ You can customize the default behavior using the following additional option:
 
   There is more detail on this option in the following section.
 
+* `-enable-plugin-cache` -  Enable the usage of the [globally configured plugin cache](/terraform/cli/config/config-file#provider-plugin-cache).
```

Comment:
would be good to just add a clarification as to why it doesn't do this automatically (or in the same way the `init` command does).